templates/CMakeLists.txt.template

```diff
@@ -282,6 +283,9 @@
   if(MSVC)
     include(cmake/msvc_static_runtime.cmake)
     add_definitions(-D_WIN32_WINNT=0x600 -D_SCL_SECURE_NO_WARNINGS -D_CRT_SECURE_NO_WARNINGS -D_WINSOCK_DEPRECATED_NO_WARNINGS)
+    if (gRPC_BUILD_MSVC_MP)
+      set(_gRPC_C_CXX_FLAGS "<%text>${_gRPC_C_CXX_FLAGS}</%text> /MP")
```

Comment:
Note that this can go fairly wild with the CPU and memory usage.

Reply:
To address this, I updated this to accept the max number of cpus to be used and this can be filled by the existing variable of `--job`.